operations/firmware/FAS_Admin_Procedures.md

```diff
@@ -8,17 +8,17 @@ Procedures for leveraging the Firmware Action Service (FAS) CLI to manage firmwa
 
 ### Topics
 
-- [FAS Admin Procedures](#fas-admin-procedures)
-  - [Topics](#topics)
-  - [Warning for Non-Compute Nodes (NCNs)</a>](#warning-for-non-compute-nodes-ncns)
-  - [Ignore Nodes within FAS](#ignore-nodes-within-fas)
-    - [Procedure](#procedure)
-  - [Override an Image for an Update](#override-an-image-for-an-update)
-  - [Procedure](#procedure-1)
-  - [Check for New Firmware Versions with a Dry-Run](#check-for-new-firmware-versions-with-a-dry-run)
-    - [Procedure](#procedure-2)
-  - [Load Firmware from Nexus](#load-firmware-from-nexus)
-  - [Load Firmware from RPM or ZIP file](#load-firmware-from-rpm-or-zip-file)
+* [FAS Admin Procedures](#fas-admin-procedures)
+    * [Topics](#topics)
+    * [Warning for Non-Compute Nodes (NCNs)</a>](#warning-for-non-compute-nodes-ncnsa)
+    * [Ignore Nodes within FAS](#ignore-nodes-within-fas)
+      * [Procedure](#procedure)
+    * [Override an Image for an Update](#override-an-image-for-an-update)
+    * [Procedure](#procedure-1)
+    * [Check for New Firmware Versions with a Dry-Run](#check-for-new-firmware-versions-with-a-dry-run)
+      * [Procedure](#procedure-2)
+    * [Load Firmware from Nexus](#load-firmware-from-nexus)
+    * [Load Firmware from RPM or ZIP file](#load-firmware-from-rpm-or-zip-file)
 
 ---
 
```

Comment:
And need to remove the errant </a> in the header on line 27

Reply:
Although I think some of my changes requested here are already covered in https://github.com/Cray-HPE/docs-csm/pull/1471/files , it occurs to me.

Probably will be fastest to get one of the two approved and merged, then rebase the other

Reply:
That sounds good to me. I intentionally omitted some of the FAS changes that were covered in that other pull request. I should have left a comment about that. Thanks for the help!